protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -598,6 +603,77 @@ message PositionUpdates{
     repeated vega.Position positions = 1;
 }
 
+message LedgerEntryFilter {
+    bool close_on_account_ids = 1;
+    repeated string account_from_ids = 2;
+    repeated string account_to_ids = 3;
+    bool close_on_account_filters = 4;
+    repeated AccountFilter account_from_filters = 5;
+    repeated AccountFilter account_to_filters = 6;
+    bool close_on_account_ids_filters = 7;
+    bool close_on_accounts_transfer_type = 8;
+    bool close_on_transfer_types = 9;
+    repeated vega.TransferType account_from_transfer_types = 10;
+    repeated vega.TransferType account_to_transfer_types = 11;
```

Comment:
Yes, this setting, as pointed in the filter comments, does not make sense when alone. It would potentially useful if used combined with other filtering.
In the simple scenario I think a single transfer type will be sufficient yes, I can do it like this

Reply:
Yes, I removed transfers from both accounts sides here - https://github.com/vegaprotocol/vega/pull/6368/files#diff-7f6c5ea53fa01231a8a36ffa313993450073ba666c91f11fe0322f33a39b1b86R613-R623